phantomgames/Search/SearchGameViewController.swift

```diff
@@ -71,12 +69,12 @@ extension SearchGameViewController: UITableViewDelegate, UITableViewDataSource {
                 CustomTableViewCell else {
             return UITableViewCell()
         }
-        let inSearchMode = viewModel.inSearchMode(searchController)
-        
-        let game = inSearchMode ? viewModel.filteredGames[indexPath.row] :
-        viewModel.allGameList[indexPath.row]
+        let newGame = viewModel.filteredGame(index: indexPath.row,
```

Comment:
indenting is off here :)
Either all on the say line or one per line under eachother 
Please re read documents we have sent to you before about coing standards :)

Reply:
this is the pr guidelines doc https://dynamicvisualtechnologies.sharepoint.com/sites/MobileInternshipProgram/SitePages/GitHub-&-SwiftLint.aspx?csf=1&web=1&e=iovxZW&cid=40132d62-11d1-4012-b03a-6d3b89942c41 . have I been looking at the wrong thing?